R/Functions_L1_Pop.R

```diff
@@ -1,4 +1,6 @@
 # ---- Level 1 Pop Functions  ----
+utils::globalVariables("colony")
+utils::globalVariables("i")
```

Comment:
I am not familiar with this type of code. What does it do? Is it for parallelisation?

Reply:
This is for building an R package. Because these variables are within the foreach package, it then complains that they are not defined.